.github/workflows/release-please.yml

```diff
@@ -0,0 +1,81 @@
+on:
+  push:
+    branches:
+      - main
+
+name: release-please
+
+jobs:
+  release-please:
+    # The secret HCLOUD_BOT_TOKEN is only available on the main repo, not in forks.
+    if: github.repository == 'ansible-collections/hetzner.hcloud'
+
+    runs-on: ubuntu-latest
+    outputs:
+      pr-updated: ${{ steps.outputs.outputs.pr-updated }}
+      branch: ${{ steps.outputs.outputs.branch }}
+      version: ${{ steps.outputs.outputs.version }}
+
+    steps:
+      - id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.HCLOUD_BOT_TOKEN }}
```

Comment:
[question][conventional comments]: What does this token do that in needs higher privileges than the default `GITHUB_TOKEN`?

[conventional comments]: https://conventionalcomments.org/

Reply:
Oh, you must be working around the limitation of the token-caused evens not propagating to other integrations like CI...

Anyway, an alternative way of doing this without a human-like machine account is to make a GitHub App and use https://github.com/marketplace/actions/create-github-app-token or https://github.com/marketplace/actions/github-app-token.

The visible interactions (like PR post/comments) would have a `[bot]` label next to the place where a typical human username appears.

Reply:
Here's a demo from my Patchback bot with how an author of such PRs/comments appears:
https://github.com/ansible-collections/community.general/pull/7201

Reply:
Thanks for the suggestion @webknjaz. If I were setting this up today I would also use a GitHub App, but @hcloud-bot predates this and the [benefits](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/differences-between-github-apps-and-oauth-apps#machine-vs-bot-accounts) are currently not worth refactoring our setup (at least to me).

Reply:
> Oh, you must be working around the limitation of the token-caused evens not propagating to other integrations like CI...

And yes, this is the purpose of the token.